xapian-applications/omega/handler_poppler.cc

```diff
@@ -23,7 +23,6 @@
 #include "handler.h"
 #include "str.h"
 
-#include <xapian.h>
```

Comment:
If xapian-core isn't used here we should also drop `$(XAPIAN_LIBS)` from `omindex_pdf_LDADD` - I've pushed that as 1060cb6ba111ed174006d262c032ce6902269019.